assets/state-gpu-admission/0300_daemonset.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: gpu-admission
+  namespace: "FILLED BY THE OPERATOR"
+spec:
+  selector:
+    matchLabels:
+      component: scheduler
+      tier: control-plane
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      serviceAccountName: gpu-admission
+      containers:
+        - image: "FILLED BY THE OPERATOR"
+          imagePullPolicy: "FILLED BY THE OPERATOR"
+          name: gpu-admission
+          env:
+            - name: LOG_LEVEL
+              value: "FILLED BY THE OPERATOR"
+          ports:
+            - containerPort: 3456
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
+      priorityClassName: system-cluster-critical
+      nodeSelector:
```

Comment:
fill nodeSelector by Operator from clusterpolicy

Reply:
@yakiduck Is this necessary? I found that the nvidia-device-plugin's nodeSelector is not configurable.

Reply:
@yakiduck I think it's because the tag's lifecycle can't be managed by the operator

Reply:
@yakiduck Maybe your configuration management can solve this problem, and it's not too late to do add this feature.

Reply:
done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/67e39ed05bf35fb8c98d9118201db2c063cf1b0a)